server/db/model.py

```diff
@@ -42,8 +111,10 @@ class User(MongoModel):
     email = fields.EmailField()
     first_name = fields.CharField()
     last_name = fields.CharField()
-    salted_password = fields.CharField()
+    salted_password = fields.CharField()  # TODO maybe add __ in the front
     gmail_oauth_info = fields.EmbeddedDocumentField(GmailOauthInfo)
+    campaigns = fields.ListField(fields.ReferenceField(Campaign, on_delete=fields.ReferenceField.PULL))
+    prospects = fields.ListField(fields.ReferenceField(Prospect, on_delete=fields.ReferenceField.PULL))
```

Comment:
Let's not put this as reference list on the user (since they will be really big) - instead let's make a field like user_id on campaigns and prospects that is indexed